Source/WebCore/editing/EditorCommand.cpp

```diff
@@ -522,7 +522,7 @@ static bool executeInsertOrderedList(LocalFrame& frame, Event*, EditorCommandSou
 
 static bool executeInsertParagraph(LocalFrame& frame, Event*, EditorCommandSource, const String&)
 {
-    TypingCommand::insertParagraphSeparator(*frame.document(), 0);
+    TypingCommand::insertParagraphSeparator(*frame.document(), OptionSet<TypingCommand::Option> { });
```

Comment:
this should be just `{ }` instead of `OptionSet<TypingCommand::Option> { }` (if it compiles)